### ISSUE TITLE
commit 799f2296fa breaks display of modules in sidebars

### ISSUE THREAD

**Issue by ralfbrown** [darktable-org/darktable#17441]
### Describe the bug

When 799f2296fa was added, all utility and processing modules which normally showed in the sidebars in lighttable, darkroom, and map views disappeared.  Darkroom still shows the nav window, scopes, modulegroup tabs, and module order, but nothing else in the sidebars.

Bisect result on master bdd838ac36 was
```
799f2296fa1a1cae3e02172cdf9b4e8dd3e78df5 is the first bad commit
commit 799f2296fa1a1cae3e02172cdf9b4e8dd3e78df5
Author: Diederik ter Rahe <dterrahe@yahoo.com>
Date:   Fri May 24 09:16:48 2024 -0400

    simplify tips when lighttable is empty
    
    add more lines pointing to correct buttons
    right align 2nd column and ellipsize if not enough space

 src/dtgtk/thumbtable.c | 187 +++++++++++++++++++++++++------------------------
 src/dtgtk/thumbtable.h |   1 +
 src/gui/gtk.c          |   1 +
 3 files changed, 97 insertions(+), 92 deletions(-)
```

### Steps to reproduce

1. Start darktable with the above-mentioned commit included; this may be with either an existing config or a new, empty config directory
2. in lighttable, observe that both sidebars are empty (no import/collections/image information on left, no history/styles/tagging/export on right)
3. if using an existing config, change to darkroom and observe no utility or processing modules in the sidebars, only nav, scopes, modulegroups header, and module order
4. switch to map view and observe empty sidebars

### Expected behavior

appropriate modules should be shown in both sidebars

### Commit

799f2296fa

### Where did you obtain darktable from?

self compiled

### darktable version

4.9.0+439~g799f2296fa

### What OS are you using?

Linux

### What is the version of your OS?

Mageia 9

### Are you using OpenCL GPU in darktable?

No

**Comment by ralfbrown** [comment]
After more fiddling, I've discovered that the modules will in fact show tooltips and respond to clicks, they just aren't visible.  Perhaps a missing gtk_widget_show{,_all} somewhere that was previously masked?  But I don't see anything in the commit which should change sidebar behavior....

**Comment by MStraeten** [comment]
reproducible with macos arm and x86 build (#17443 fixes that behaviour)

**Comment by ralfbrown** [comment]
Reported on pixls.us for the nightly MacOS build.

**Comment by ralfbrown** [comment]
Fixed by #17425.
